compair/models/mixins/uuid_mixin.py

```diff
@@ -1,6 +1,8 @@
 import uuid
 import base64
 
+from sqlalchemy.orm import joinedload
```

Comment:
added but not used?

Reply:
It is used in `get_by_uuid_or_404` but not imported. I guess no one used this function before.

Reply:
Oh your right! good catch.